lib/generators/module/templates/gemspec.erb

```diff
@@ -3,11 +3,11 @@
 $LOAD_PATH.push File.expand_path('lib', __dir__)
 
 # Maintain your gem's version:
-require '<%= class_name.downcase %>/version'
+require '<%= class_name.underscore %>/version'
 
 # Describe your gem and declare its dependencies:
 Gem::Specification.new do |spec|
-  spec.name        = '<%= class_name.downcase %>'
+  spec.name        = '<%= class_name.underscore %>'
```

Comment:
should this be downcase and underscore?

Reply:
"Makes an underscored, lowercase form from the expression in the string." https://apidock.com/rails/String/underscore

Reply:
I should have checked first!